ambientweather2sqlite/mureq.py

```
@@ -190,55 +231,66 @@ def yield_response(
+    def raise_for_status(self) -> None:
         """raise_for_status checks the response's success code, raising an
         exception for error codes.
         """
         if not self.ok:
             raise HTTPErrorStatus(self.status_code)
 
-    def json(self):
+    def json(self) -> JsonValue:
         """Attempts to deserialize the response body as UTF-8 encoded JSON."""
-        import json as jsonlib
+        import json as jsonlib  # noqa: PLC0415
```

Ruff (0.11.9)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)